src/wp-includes/class-wp-query.php

```diff
@@ -2882,7 +2882,7 @@ public function get_posts() {
 			$comments_request = "SELECT $distinct {$wpdb->comments}.comment_ID FROM {$wpdb->comments} $cjoin $cwhere $cgroupby $corderby $climits";
 
 			$key          = md5( $comments_request );
-			$last_changed = wp_cache_get_last_changed( 'comment' ) . ':' . wp_cache_get_last_changed( 'posts' );
+			$last_changed = wp_cache_get_last_changed( 'comment' ) . '+' . wp_cache_get_last_changed( 'posts' );
```

Comment:
What's the rationale for using `+` over `:`? I'm mostly asking why you're proposing to _change_ this.

Reply:
How does using `+` over `:` make it easier? I'm not sure I follow, since you always have to specify the separator for `explode()`. Can you clarify?

Reply:
I found it easier to rely on splitting by `:` will give me the query, key and timestamp(s) as 3 elements. Then the timestamp part can be broken down more. Felt cleaner than 3-4 elements depending on the query.